### DISCUSSION TITLE
FSL standard problem: LowerCingulum_1mm

### DISCUSSION THREAD

**Discussion by HellewellS** [neurodesk/neurodesk.github.io#100] [Q&A] [answered]
Hello,

I'm trying to run some TBSS analyses in Neurodesk and keep encountering a problem with what seems to be a bad image file in the FSL directory. I have tried loading this file in FSLeyes (all versions) and it fails to load and crashes. This issue is in tbss_4_prestats, where all FA data is projected onto the skeleton:
${FSLDIR}/bin/tbss_skeleton -i mean_FA -p $thresh mean_FA_skeleton_mask_dst ${FSLDIR}/data/standard/LowerCingulum_1mm all_FA all_FA_skeletonised

My error is:
Image Exception : neurodesk/neurodesk.github.io#22 :: Failed to read volume /opt/fsl-6.0.5.1/data/standard/LowerCingulum_1mm
Error : Error: cant open file /opt/fsl-6.0.5.1/data/standard/LowerCingulum_1mm.nii.gz
Same error no matter the FSL version.

Is there a way to replace this nifti in the fsl directory? Or better to run that section of the prestats command separately and point it to a local file?

Thanks!
Sarah

**Comment by stebo85** [comment]
Dear @HellewellS 

Thank you for reporting this problem! 

I currently cannot yet reprodcue the problem. Could you try the following to narrow down the issue:

open fsl 6.0.5.1 from the application menu and then run 
```
fsleyes /opt/fsl-6.0.5.1/data/standard/LowerCingulum_1mm.nii.gz 
```

This should open the file:
<img width="2528" alt="image" src="https://user-images.githubusercontent.com/4021595/215379077-fa1ce6cb-5af9-4e50-b380-33d2142ef5d9.png">

Do you get the file or do you get the error you reported?

Thank you :)
Steffen

**Reply by HellewellS** [reply]
Hi Steffen,

It crashes when trying to open from the terminal, with the message:
You can get a copy from:
	http://icon-theme.freedesktop.org/releases
Segmentation fault

Similarly to the GUI, it tries to load, opening FSLeyes and then crashes out.

Thanks,
Sarah

**Reply by stebo85** [reply]
Dear @HellewellS,

ok, perfect. 

Another few questions to narrow down the problem further:
Can you restart your neurodesktop session and does the problem still persist?
Which version of neurodesktop are you using (e.g. vnmd/neurodesktop:20221216 ?)
Are you running docker on Windows, Linux or Macos?

Thank you 
Steffen

**Comment by HellewellS** [comment, marked as answer]
Dear Steffen,

Restarting neurodesktop did the trick. I am running on docker on Windows, using neurodesktop version 20221216.

Thanks for your help! :)

**Reply by stebo85** [reply]
Ok, great to hear it is all working now as expected after restarting.
